### PR TITLE
[DotNetCore] Enable "Publish to folder" command for all .NET Core projects

### PR DESCRIPTION
**Pull request by rodrmoya**
Fixes https://devdiv.visualstudio.com/DevDiv/_workitems/edit/945334

### CODE REVIEW

File: main/src/addins/MonoDevelop.DotNetCore/MonoDevelop.DotNetCore/TargetFrameworkExtensions.cs [hunk 1]
```diff
@@ -91,6 +91,8 @@ public static bool IsNetCoreAppOrHigher (this TargetFramework framework, DotNetC
 
 		public static bool IsNetFramework (this TargetFramework framework) => framework.Id.IsNetFramework ();
 
+		public static bool IsNetStandardOrNetCore (this TargetFramework framework) => framework.IsNetStandard () || framework.IsNetCoreApp ();
```
Comment (jtorres) [inline main/src/addins/MonoDevelop.DotNetCore/MonoDevelop.DotNetCore/TargetFrameworkExtensions.cs]: we already have such extension here https://github.com/mono/monodevelop/blob/9766b935ea3541e80e85f88a0ed7d64253bcb6f5/main/src/addins/MonoDevelop.DotNetCore/MonoDevelop.DotNetCore/TargetFrameworkMonikerExtensions.cs#L48

Comment (rodrmoya) [reply main/src/addins/MonoDevelop.DotNetCore/MonoDevelop.DotNetCore/TargetFrameworkExtensions.cs]: ugh, I promise I even looked for it, but I guess I missed it :) fixing...

Comment (rodrmoya) [reply main/src/addins/MonoDevelop.DotNetCore/MonoDevelop.DotNetCore/TargetFrameworkExtensions.cs]: Fixed now

Comment (rodrmoya) [reply main/src/addins/MonoDevelop.DotNetCore/MonoDevelop.DotNetCore/TargetFrameworkExtensions.cs]: I'll have a look at improving the 2 *Extensions classes in a future PR

**Pull request merged**